llvm/lib/Target/DirectX/DirectX.h

```diff
@@ -58,6 +58,12 @@ void initializeDXILPrettyPrinterPass(PassRegistry &);
 /// Initializer for dxil::ShaderFlagsAnalysisWrapper pass.
 void initializeShaderFlagsAnalysisWrapperPass(PassRegistry &);
 
+/// Initializer for dxil::DXILMetadataAnalysisLegacy pass.
+void initializeDXILMetadataAnalysisLegacyPassPass(PassRegistry &);
+
+/// Initializer for dxil::DXILMetadataAnalysisLegacyPrinter pass.
+void initializeDXILMetadataAnalysisLegacyPrinterPass(PassRegistry &);
+
```

Comment:
These are declared in `include/llvm/InitializePasses.h`, they shouldn't be here as well.